src/components/permissions/api/PermissionsAPI.web.js

```diff
@@ -1,27 +1,28 @@
 // @flow
-import { invokeMap, isFunction } from 'lodash'
+import { assign, invokeMap, isFunction } from 'lodash'
+import { isSupported } from 'firebase/messaging'
 import Config from '../../../config/config'
 
 import logger from '../../../lib/logger/js-logger'
 import { type Permission, Permissions, type PermissionStatus, PermissionStatuses } from '../types'
 
+const { Granted, Undetermined, Denied, Disabled, Prompt } = PermissionStatuses
 const { enableWebNotifications } = Config
+const toStatusEnum = status => ([Granted, Denied].includes(status) ? status : Undetermined)
 class PermissionsAPI {
   // permissions enum to platform permissions map
   platformPermissions = {
     [Permissions.Camera]: 'camera',
     [Permissions.Clipboard]: 'clipboard-read',
+    [Permissions.Notifications]: 'notifications',
```

Comment:
in this PR should be disabled. check this commit (latest before web support been added) https://github.com/GoodDollar/ScheduledNotifications/commit/1dd57585c15c6f3e18fa36d9bb22e83f8b1db04b